forms/uds/a1/form_a1_ivp_error_checks_p.csv

```diff
@@ -0,0 +1,52 @@
+error_code,error_no,error_type,form_name,packet,var_name,check_type,test_name,short_desc,full_desc,test_logic,comp_forms,comp_vars,do_in_redcap,in_prev_versions,questions
+a1-ivp-p-1001,1001,Alert,a1,I,RACEWHITE,Plausibility,"If one race is known, cannot check Unknown","If  RACEWHITE, RACEBLACK, RACEAIAN, RACENHPI, RACEASIAN, ETHISPANIC, or RACEOTH=1 then RACEUNKN cannot equal 1","If  Q2a. RACEWHITE, Q2b. RACEBLACK, Q2c. RACEAIAN, Q2d. RACENHPI, Q2e. RACEASIAN, Q2f. ETHISPANIC, or Q2g. RACEOTH=1 then Q2h. RACEUNKN cannot equal 1",IF (RACEWHITE=1 or RACEBLACK=1 or RACEAIAN=1 or RACENHPI=1 or RACEASIAN=1 or ETHISPANIC=1 or RACEOTH=1) and RACEUNKN=1,,"RACEWHITE,
+RACEBLACK,
+RACEAIAN,
+RACENHPI,
+RACEASIAN,
+ETHISPANIC,
+RACEOTH,
+RACEUNKN",No,No,
+a1-ivp-p-1002,1002,Alert,a1,I,GENMAN,Plausibility,"If one gender is known, cannot check Don't know","If  GENMAN, GENWOMAN, GENTRMAN, GENTRWOMAN, GENNONBI, GENTWOSPIR, GENOTH, or GENNOANS=1 then GENDKN cannot equal 1","If  Q3a. GENMAN, Q3b. GENWOMAN, Q3c. GENTRMAN, Q3d. GENTRWOMAN, Q3e. GENNONBI, Q3f. GENTWOSPIR, Q3g. GENOTH, or Q3i. GENNOANS=1 then Q3hg. GENDKN cannot equal 1",IF (GENMAN=1 or GENWOMAN=1 or GENTRMAN=1 or GENTRWOMAN=1 or GENNONBI=1 or GENTWOSPIR=1 or GENOTH=1 or GENNOANS=1) and GENDKN=1,,"GENMAN,
+GENWOMAN,
+GENTRMAN,
+GENTRWOMAN, GENNONBI,
+GENTWOSPIR,
+GENOTH,
+GENDKN,
+GENNOANS",No,No,
+a1-ivp-p-1003,1003,Alert,a1,I,GENMAN,Plausibility,"If one gender is known, cannot check No answer","If  GENMAN, GENWOMAN, GENTRMAN, GENTRWOMAN, GENNONBI, GENTWOSPIR, GENOTH, or GENDKN=1 then GENNOANS cannot equal 1","If  Q3a. GENMAN, Q3b. GENWOMAN, Q3c. GENTRMAN, Q3d. GENTRWOMAN, Q3e. GENNONBI, Q3f. GENTWOSPIR, Q3g. GENOTH, or Q3h. GENDKN=1 then Q3i. GENNOANS cannot equal 1",IF (GENMAN=1 or GENWOMAN=1 or GENTRMAN=1 or GENTRWOMAN=1 or GENNONBI=1 or GENTWOSPIR=1 or GENOTH=1 or GENDKN=1) and GENNOANS=1,,"GENMAN,
```

Comment:
Please change the var_name for error a1-ivp-p-1003 to "GENNOANS", it's easy to implement this check under that variable.